tftrt/examples/image-classification/image_classification.py

```diff
@@ -52,29 +52,34 @@ def after_run(self, run_context, run_values):
                 current_step, self.num_steps, duration * 1000,
                 self.batch_size / self.iter_times[-1]))
 
-class DurationHook(tf.train.SessionRunHook):
+class BenchmarkHook(tf.train.SessionRunHook):
     """Limits run duration"""
-    def __init__(self, target_duration):
+    def __init__(self, target_duration=None, iteration_limit=None):
         self.target_duration = target_duration
         self.start_time = None
+        self.iteration_number = 0
```

Comment:
Could you please rename `iteration_number` to `current_iteration`?

Reply:
sure